src/main/java/com/synopsys/integration/jenkins/scan/service/scan/ScanStrategyService.java

```diff
@@ -0,0 +1,12 @@
+package com.synopsys.integration.jenkins.scan.service.scan;
+
+import com.synopsys.integration.jenkins.scan.global.enums.ScanType;
+import java.util.Map;
+
+public interface ScanStrategyService {
```

Comment:
This class **shouldn't** be inside `service.scan` package. Rather move it to another package named `strategy` which would be the same directory level of `service`. 
Also rename the interface name as **ScanStrategy** only.

Reply:
Packaged and updated accordingly